bitbots_dynamic_kick/src/visualizer.cpp

```diff
@@ -78,4 +80,47 @@ void Visualizer::displayWindupPoint(const tf2::Vector3 &kick_windup_point, const
   windup_publisher_.publish(marker);
 }
 
+void Visualizer::publishGoals(const KickPositions &positions,
+                              const KickPositions &stabilized_positions,
+                              const robot_state::RobotStatePtr &robot_state) {
+  std::string support_foot_frame, flying_foot_frame;
```

Comment:
you could check first if there are subscribers before spending CPU time on this

//only do something if someone is listing
if (debug_publisher_.getNumSubscribers() == 0) {
  return;
}

Reply:
Thanks, I added it.